recipes/mne-bids-pipeline/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "mne-bids-pipeline" %}
+{% set version = "1.0.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/mne-bids-pipeline-{{ version }}.tar.gz
+  sha256: 6cdc9864d27d5a5d56f1204aef3fe9e49078e75be08cb74ff5e5d3253b151e62
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  noarch: python
+  entry_points:
+    - mne_bids_pipeline = mne_bids_pipeline._main:main
+
+requirements:
+  host:
+    - pip
+    - setuptools >=65.0
+    - python >=3.8
+  run:
+    - autoreject
+    - bokeh <3
+    - coloredlogs
+    - dask-core
```

Comment:
I think we also need `distributed` but I'm not entirely sure, could you check please?

Reply:
I'll just replace with `dask` which takes care of bokeh + distributed